var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -29,6 +29,8 @@ class Dealii(Package):
     depends_on ("boost@1.59.0:+mpi", when='+mpi')
     depends_on ("bzip2")
     depends_on ("cmake")
+    depends_on ("gls", when='@8.5.0:')
```

Comment:
Cool, thanks @davydden . I think it keeps with deal.II's philosophy of offering a near bare-bones library and all of the extensions are optional. I reckon that we should try to do the same wherever possible (especially since it doesn't take much effort at all :-).